docs/DeveloperGuide.adoc

```diff
@@ -763,18 +822,55 @@ Use case ends.
 * 3a. The given index is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. CYNC shows an error message.
++
+Use case resumes at step 2.
+
+* 4a. Deletion not successful
++
+[none]
+** 4a1. CYNC shows an error messages
++
+Use case resumes at step 2
+
+=== Edit Customer's details
+
+*MSS*
+
+1. Customer Manager requests to edit customer
```

Comment:
How does the customer manager identify the customer to edit? Is it searching by the name, or specifying the index in the last listing (same as in your UserGuide)? For alternative, he might need to search for the particular customer so that it's listed out before step 1 can be performed